src/components/NcSelect/NcSelect.vue

```diff
@@ -1007,6 +1007,36 @@ body {
 			}
 		}
 	}
+
+	// Hide search from dom if unused to prevent unneeded flex wrap
+	.vs__search[readonly] {
+		position: absolute;
+	}
+	// If search if hidden, ensure that the height of the search is the same
+	.vs__selected-options {
+		min-height: 40px; // 36px search height + 4px search margin
+	}
+
+	/**
+	 * Fix overlow of selected options
+	 * There is an upstream pull request, if it is merged and released remove this fix
+	 * https://github.com/sagalbot/vue-select/pull/1756
+	 */
+	.vs__selected-options {
+		min-width: 0;
+		.vs__selected {
+			min-width: 0;
+		}
+	}
```

Comment:
I just noticed that these two lines create issues when the `NcSelect` component has `:no-wrap="true"`set. See here:
![Screenshot 2023-02-22 at 12-45-19 Nextcloud Vue Style Guide](https://user-images.githubusercontent.com/2496460/220611641-f2789437-48d1-4850-b4ee-cde8dc4339bd.png)

Reply:
@susnux I created a new issue for this: https://github.com/nextcloud/nextcloud-vue/issues/3808

Reply:
Mh, shall we revert the PR for this release @raimund-schluessler?

Reply:
I don't really have a preference here.

Reply:
I don't think there's a need for rush, so let's try to get this solved before releasing 7.7.0. @susnux would you prefer to look for a fix or to revert the PR for 7.7.0?

Reply:
I will look for a fix.